src/Avalonia.Themes.Fluent/Accents/AccentColors.xaml.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+using Avalonia.Markup.Xaml;
+using Avalonia.Media;
+using Avalonia.Platform;
+using Avalonia.Styling;
+
+namespace Avalonia.Themes.Fluent.Accents
+{
+    public class AccentColors : Styles
+    {
+        public AccentColors()
+        {
+            AvaloniaXamlLoader.Load(this);
+
+            var accentColor_prov = AvaloniaLocator.CurrentMutable.GetService<IPlatformAccentColorProvider>();
+
+            var accentcolor = accentColor_prov == null ? Color.Parse("#FF0078D7") : accentColor_prov.AccentColor;
```

Comment:
if the provider is null it should read the one from the resource.

Reply:
@PieroCastillo can you move "#FF0078D7" to the resources and name it something like "FallbackSystemAccentColor"?

Reply:
Note this fallback color most likely shouldn't be set in the same file as where you include AccentColors style, since that file initialization wasn't yet completed in moment of AccentColors.ctor, and you won't be able to read another resources from that file.